aws_google_auth/google.py

```diff
@@ -206,8 +209,72 @@ def do_login(self):
         self.check_extra_step(response_page)
 
         if cap is not None:
-            raise ExpectedGoogleException(
-                'Captcha Required. Manually Login to remove this.')
+            self.session.headers['Referer'] = sess.url
+
+            # Collect ProfileInformation, SessionState, signIn, and Password Challenge URL
+            challenge_page = response_page
```

Comment:
I understand the desire to rename `response_page` to `challenge_page`, but maybe this could also be solved by breaking all this off into it's own function (`handle_captcha()`), like I mentioned somewhere else. Then this is less of an issue.